python/paddle/fluid/clip.py

```diff
@@ -282,7 +282,6 @@ def _create_operators(self, param, grad):
         if group_scale_name not in self.context:
             group_norm_var = layers.sums(input=self.context[self.group_name])
             group_norm_var = layers.sqrt(x=group_norm_var)
-            group_norm_var = layers.cast(group_norm_var, 'float32')
```

Comment:
This line is added by @ktlichkid in this PR https://github.com/PaddlePaddle/Paddle/pull/13516, I think this is because we didn't set the DataType in Activation_op, so the data type of `layers.sqrt(x=group_norm_var)` is BOOL, is it right? @ktlichkid
Now I have fixed it [here](https://github.com/PaddlePaddle/Paddle/pull/13723/files#diff-b6a56915c851e097c696f5ef8a5724cfR102).

Reply:
This is to cast temporary float64 (to prevent overflow) value back to float32.